docs/advanced/local_extensions.rst

```diff
@@ -0,0 +1,36 @@
+.. _`template extensions`:
+
+Local Extensions
+----------------
+
+*New in Cookiecutter X.x*
+
+A template may extend the Cookiecutter environment with local extensions.
+These can be part of the template itself, providing it with more sophisticated custom tags and filters.
+
+To do so, a template author must specify the required extensions in ``cookiecutter.json`` as follows:
+
+.. code-block:: json
+
+    {
+        "project_slug": "Foobar",
+        "year": "{% now 'utc', '%Y' %}",
+        "_local_extensions": ["local_extensions.FoobarExtension"]
+    }
+
+This example assumes that a ``local_extensions`` folder (python module) exists in the template root.
+It will contain a ``main.py`` file, containing the following (for instance):
```

Comment:
If I only need one or two extensions and they're not that complicated, I can just define them directly in a module `local_extensions.py`, correct? Maybe instead of having the documentation suggest a more-complicated package setup, it could suggest simply that you have a module `local_extensions` available in the template root with extensions defined, with no reference to a folder or `main.py`. That would leave it up to the user to use a standalone module or a package structure depending on their needs.

Reply:
that seems fair. i'll make the change

Reply:
From what I gather reading the code you are adding the path of the local directory. So in effect you could create a folder that is named `extensions` that contains an `__init__.py` file. In that file you'll have the extensions and you'd refer to that in the cookiecutter.json file as `'extensions.MyExtension'` and it should all work.